typings/index.d.ts

```diff
@@ -89,6 +89,17 @@ declare module 'discord.js' {
     public valueOf(): string;
   }
 
+  export class BaseEmoji {
```

Comment:
```suggestion
  export class BaseEmoji extends Emoji {
```
this class extends emoji

imo though, i think Emoji should extend BaseEmoji